src/rai/rai/cli/rai_cli.py

```diff
@@ -35,42 +41,80 @@ def parse_whoami_package():
+            "as well as the robot's capabilities and limitations including sensor and actuator information. "
+            "If there are any images provided, make sure to take them into account by thoroughly analyzing them. "
+            "Your description should be thorough and detailed."
+            "Your reply should start with I am a ..."
+        )
+        llm = ChatOpenAI(model="gpt-4o-mini")
+
+        images = glob.glob(args.documentation_root + "/images/*")
+
+        messages = [SystemMessage(content=prompt)] + [
+            HumanMultimodalMessage(
+                content=documentation,
+                images=[preprocess_image(image) for image in images],
+            )
+        ]
+        output = llm.invoke(messages)
+        assert isinstance(output.content, str), "Malformed output"
+
+        with open(save_dir + "/robot_identity.txt", "w") as f:
+            f.write(output.content)
+        logger.info("Done")
 
     docs = ingest_documentation(
         documentation_root=args.documentation_root + "/documentation"
     )
-    faiss_index = FAISS.from_documents(docs, OpenAIEmbeddings())
-    faiss_index.add_documents(docs)
-    save_dir = args.output
-    faiss_index.save_local(save_dir)
-
-    prompt = (
-        "You will be given a robot's documentation. "
-        "Your task is to identify the robot's identity. "
-        "The description should cover the most important aspects of the robot with respect to human interaction, "
-        "as well as the robot's capabilities and limitations including sensor and actuator information. "
-        "If there are any images provided, make sure to take them into account by thoroughly analyzing them. "
-        "Your description should be thorough and detailed."
-        "Your reply should start with I am a ..."
+    documentation = str([doc.page_content for doc in docs])
+    n_tokens = len(documentation) // 4.0
+    logger.info(
+        f"The documentation's length is {len(documentation)} chars, "
+        f"approximately {n_tokens} tokens"
     )
-    llm = ChatOpenAI(model="gpt-4o-mini")
-
-    images = glob.glob(args.documentation_root + "/images/*")
-
-    messages = [SystemMessage(content=prompt)] + [
-        HumanMultimodalMessage(
-            content=str([doc.page_content for doc in docs]),
-            images=[preprocess_image(image) for image in images],
+    logger.warn(
+        f"Building the robot docs vector store will cost "
+        f"approximately {n_tokens / 1_000_000 * 0.1:.4f}$. "
+        "Do you want to continue? (y/n)"
+    )
+    if input() == "y":
+        build_docs_vector_store()
+    else:
+        logger.info("Skipping the robot docs vector store creation.")
+
+    logger.warn(
+        f"Building the robot identity will cost "
+        f"approximately {n_tokens / 1_000_000 * 0.15:.4f}$. "
+        "Do you want to continue? (y/n)"
+    )
```

Comment:
Please mention in the warning that it can be done manually (copy the logger.info message to the warning)

Reply:
Done